backend/package-lock.json

```diff
@@ -1,1818 +1,8 @@
 {
   "name": "near-explorer-backend",
   "version": "0.0.1",
-  "lockfileVersion": 2,
```

Comment:
Please, revert the changes in the `backend/package-lock.json` since they are no-op and not relevant to this PR anyway.